database/seeds/PostsTableSeeder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class PostsTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+      $faker = Faker\Factory::create();
+
+      for ($i = 0; $i < 10; $i++){
+          DB::table('posts')->insert([ //,
+              'url' => $faker->unique()->word,
+              'title' => $faker->unique()->sentence($nbWords = 6),
+              'description' => $faker->paragraph($nbSentences = 1),
+              'content' => $faker->text,
+              'image' => $faker->randomElement($array = array ('blog-one.jpg','blog-two.jpg','blog-three.jpg')),
+              'blog' => '1',
+              'category_id' => $faker->numberBetween($min = 1, $max = 3),
```

Comment:
This is cheat code. You should select `category_id` from `categories` table. 
` 'category_id' => $faker->randomElement($categoryId ?: $categoryId = App\Category::pluck('id')->toArray()),`